bot/resources/tags/empty-json.md

```diff
@@ -0,0 +1,27 @@
+When creating a new JSON file you might run into the following error.
+
+`JSONDecodeError: Expecting value: line 1 column 1 (char 0)`
+
+In short this error means your JSON is invalid in it's current state.
+A JSON may never be completely empty and must always at least have one of the following items.
+
+```
+object
+array
+string
+number
+"true"
+"false"
+"null"
+```
+
+To resolve this issue, you create one of the above values in your JSON. It is very common to use `{}` to make an object. Adding the following to your JSON should resolve this issue.
+
+```json
+{
+
+
+}
```

Comment:
Does this mean that the JSON in this code example, if saved exactly as-is in a .json file, would be valid because it has the extra line? That's my takeaway but that seems to conflict with what else is conveyed in this message.

Reply:
That extra white line between the brackets is removed.
As stated earlier when it's COMPLETELY empty (like when it's just created) it's invalid. But this makes it have at least SOME content to make it valid.
Mayb there is a better way to word it? or maybe an extra note to add?